src/main/java/io/github/thebusybiscuit/slimefun4/implementation/guide/SurvivalSlimefunGuide.java

```diff
@@ -533,19 +534,35 @@ private void displayItem(ChestMenu menu, PlayerProfile profile, Player p, Object
         MenuClickHandler clickHandler = (pl, slot, itemstack, action) -> {
             try {
                 if (itemstack != null && itemstack.getType() != Material.BARRIER) {
-                    displayItem(profile, itemstack, 0, true);
+                        displayItem(profile, itemstack, 0, true);
                 }
             } catch (Exception | LinkageError x) {
                 printErrorMessage(pl, x);
             }
             return false;
         };
 
+        MenuClickHandler ununlockedClickHandler = (pl, slot, itemstack, action) -> {
```

Comment:
UnUnlocked? You mean "locked"?

Reply:
Yeah I was probably asleep when I named it that way